rpcserver.go

```diff
@@ -5718,6 +5728,17 @@ func handleVerifyMessage(s *rpcServer, cmd interface{}, closeChan <-chan struct{
 	return address.EncodeAddress() == c.Address, nil
 }
 
+// handleVersion implements the version command.
+func handleVersion(s *rpcServer, cmd interface{}, closeChan <-chan struct{}) (interface{}, error) {
+	result := &dcrjson.VersionResult{
+		VersionString: jsonrpcSemverString,
+		Major:         jsonrpcSemverMajor,
+		Minor:         jsonrpcSemverMinor,
+		Patch:         jsonrpcSemverPatch,
+	}
```

Comment:
To me it seems like these would only exist to be changed at link time.  If they can change, then the version string would also need to be dynamically created.  I can do that, but it's a little more work for a situation that we will probably never end up needing.